packages/block-editor/src/components/block-list/block-list-item-cell.native.js

```diff
@@ -7,14 +7,23 @@ import { View } from 'react-native';
  * WordPress dependencies
  */
 import { useEffect, useCallback } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
 import { useBlockListContext } from './block-list-context';
+import { store as blockEditorStore } from '../../store';
 
-function BlockListItemCell( { children, clientId, rootClientId, onLayout } ) {
+function BlockListItemCell( { children, item: clientId, onLayout } ) {
 	const { blocksLayouts, updateBlocksLayouts } = useBlockListContext();
+	const { rootClientId } = useSelect(
+		( select ) => {
+			const { getBlockRootClientId } = select( blockEditorStore );
+			return { rootClientId: getBlockRootClientId( clientId ) };
+		},
+		[ clientId ]
+	);
```

Comment:
React Native `VirtualizedList` expects a [React class component or render function](https://reactnative.dev/docs/0.69/virtualizedlist#cellrenderercomponent). Passing a function that returned a React Element resulted in an inability to open the inserter from a inner block appender — multiple re-renders kept resetting `isOpen`. 

In order to pass the `BlockListItemCell` component, the `rootClientId` had to be selected here instead. It does cause me concern as to whether this is will degrade performance.

I welcome thoughts on performance or possible alternative approaches.